build-system/tasks/dist.js

```diff
@@ -55,6 +60,37 @@ const {maybeUpdatePackages} = require('./update-packages');
 const {green, cyan} = colors;
 const argv = require('minimist')(process.argv.slice(2));
 
+const babel = require('@babel/core');
+const deglob = require('globs-to-files');
+
+function transferSrcsToTempDir() {
+  const tmp = `${tempy.directory()}/amphtml`;
+  process.env.AMP_TMP_DIR = tmp;
+
+  if (!isTravisBuild()) {
+    log('Transforming JS files to', cyan(tmp));
+  }
+  const files = deglob.sync(BABEL_SRC_GLOBS);
+  log(green('Executing babel transforms'));
+  files.forEach(file => {
+    if (file.startsWith('node_modules/') || file.startsWith('third_party/')) {
+      fs.copySync(file, `${tmp}/${file}`);
+      return;
+    }
+
+    const {code} = babel.transformFileSync(file, {
+      plugins: conf.plugins({
+        isEsmBuild: argv.esm,
+        isForTesting: argv.fortesting,
+      }),
+      retainLines: true,
+      compact: true,
```

Comment:
To kill the useless warning, it should be `compact: false`, not `compact: true`.

Reply:
woops. fixed.